### PR TITLE
About us Section Contents and bg image added

### PR DESCRIPTION
**Pull request by Anand-Madhu123**
linkbrain
a6

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -138,16 +138,37 @@ <h2 class="pa-h2-v2-hero-subhead pa-white">Let us do the technological and profe
...
+    <div class="container">
+        <div class="row">
+            <div class="col-sm-6">
+                <div class="container">
+                   <h1> We make our <br>people and products<br>sensational.</h1>
+                            <br>
+                            <p>To revolutionize the way people do software and delivering happiness to our stakeholders and society.</p>
+                            <br>
+                        </div>
+                        <a href="https://themeforest.net/item/particle-modern-tech-startup-html-template/20078383?ref=Skilltech" target="_blank">Schedule Your Free Consultation &nbsp;</a>
```
Comment (wandercode) [inline index.html]: please make it to a proper button like 
image

**Review by wandercode — CHANGES_REQUESTED** [review]

**Comment by wandercode** [comment]
LGTM 👍

**Review by wandercode — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -138,16 +138,38 @@ <h2 class="pa-h2-v2-hero-subhead pa-white">Let us do the technological and profe
...
+                   <h1> We make our <br>people and products<br>sensational.</h1>
+                            <br>
+                            <p>To revolutionize the way people do software and delivering happiness to our stakeholders and society.</p>
+                            <br>
+                        </div>
+                        <a class="pa-button-secondary pa-hvr-sweep-to-right fadeInRightDelay11Duration10" href="" target="_blank">Schedule Your Free Consultation</a>    
+                    </div>
+            <div class="col-sm-6">
+               <br><br><br>
+                <p class="content">We are public benefit organization established when the world appears to be on cross roads in light of COVID-19 pandemic crisis.
```
Comment (VishnuKulakkattu) [inline index.html]: try to arrange the Header text and the Paragraph in equal line.
image

**Review by VishnuKulakkattu — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Anand-Madhu123) [reply index.html]: done

File: index.html (hunk 2 above)
Comment (Anand-Madhu123) [reply index.html]: done

**Review by rohithkrajan — CHANGES_REQUESTED** [review]
remove not needed images

File: index.html [hunk 3]
```diff
@@ -138,16 +138,36 @@ <h2 class="pa-h2-v2-hero-subhead pa-white">Let us do the technological and profe
...
+
+    <div class="container">
+        <div class="row">
+            <div class="col-sm-6">
+                <br>    
+                   <h1> We make our <br>people and products<br>sensational.</h1>
+                            <br>
+                            <p>To revolutionize the way people do software and delivering happiness to our stakeholders and society.</p>
+                            <br>
+                         <a class="pa-button-secondary pa-hvr-sweep-to-right fadeInRightDelay11Duration10" href="" target="_blank">Schedule Your Free Consultation </a>
```
Comment (rohithkrajan) [inline index.html]: We should link this to https://brainchildschool.com/partner-with-us

File: index.html [hunk 4]
```diff
@@ -138,16 +138,36 @@ <h2 class="pa-h2-v2-hero-subhead pa-white">Let us do the technological and profe
...
-    <!-- CTA Section
-	================================================== -->
-    <section class="pa-cta-v1" data-parallax="scroll" data-speed="0.2" data-image-src="assets/images/image-section.jpg">
-
-        <div class="container">
-            <h1 class="pa-h1-v1 pa-bright fadeInLeftDelay03Duration10"><strong>Coolest HTML Theme Yet?</strong></h1>
-            <h4 class="pa-h4-v1 pa-bright fadeInLeftDelay03Duration10">Open Particle today, why not!</h4>
-            <div class="mt-4"></div>
-            <a class="pa-button-primary pa-hvr-sweep-to-right fadeInRightDelay11Duration10" href="https://themeforest.net/item/particle-modern-tech-startup-html-template/20078383?ref=Skilltech" target="_blank">Get it Now</a>
+	<section class="pa-cta-v1" data-parallax="scroll" data-speed="0.2" data-image-src="assets/images/web5.jpg">
```
Comment (rohithkrajan) [inline index.html]: Please change from web5 to a meaning ful name image-aboutus.jpg

**Review by rohithkrajan — CHANGES_REQUESTED** [review]
LGTM

File: index.html (hunk 4 above)
Comment (Anand-Madhu123) [reply index.html]: done

Comment (Anand-Madhu123) [reply index.html]: done

File: index.html (hunk 3 above)
Comment (Anand-Madhu123) [reply index.html]: done

**Review by VishnuKulakkattu — APPROVED** [review]

**Review by rohithkrajan — APPROVED** [review]

**Pull request merged**